inspire/modules/formatter/views.py

```diff
@@ -84,3 +88,36 @@ def get_latex_file(recid, latex_format):
             )
         }
     )
+
+
+@blueprint.route("/export-as/<path:export_format>/<path:ids>",
```

Comment:
This function receives a list of `ids` specified by the user and returns all the given formats.

We should be defensive with the user input and:

* Set a limit to the `ids` list. Or we have to explore if using Celery we can scale and be able to export as many `ids` as requested.
* What about user permissions? Do we have hidden records for certain users? Should we do the intersection of the requested ids with the ids the user is allowed to see? Is there an API to get all records allowed by a user?

What do you think @kaplun @jalavik  ? Something else?